### ISSUE TITLE
Older D-Bus version

### ISSUE THREAD

**Issue by taheris** [advancedtelematic/aktualizr#104]
As it stands we currently make use of [dbus_message_iter_get_element_count](https://github.com/advancedtelematic/aktualizr/blob/7446be358315f415a3a2fb50c7faea4bdfccb00f/src/dbusgateway/dbusgateway.cc#L139) which was only merged as of D-Bus [1.9.16](https://launchpad.net/debian/+source/dbus/1.9.16-1).

We should target (as a maximum) the version of D-Bus in Debian Jessie which is 1.8.22, or ideally the prior stable Debian Wheezy version of 1.6.8.

**Comment by cajun-rat** [comment]
When making this change, we should change the Docker file to point to Debian Jessie too.

**Comment by taheris** [comment]
I've created a Dockerfile [here](https://github.com/advancedtelematic/aktualizr/blob/feat/104/debian-jessie/Dockerfile) that does this, although it won't build yet for the above reason.

**Comment by cajun-rat** [comment]
I've found another place that stops the build on Debian Stable:

~~~~~
/aktualizr/src/src/dbusgateway/dbusgateway.cc: In member function 'data::OperationResult DbusGateway::getOperationResult(DBusMessageIter*)':
/aktualizr/src/src/dbusgateway/dbusgateway.cc:180:56: error: 'dbus_message_iter_get_element_count' was not declared in this scope
   int params = dbus_message_iter_get_element_count(iter);
                                                        ^
CMakeFiles/aktualizr_static_lib.dir/build.make:330: recipe for target 'CMakeFiles/aktualizr_static_lib.dir/src/dbusgateway/dbusgateway.cc.o' failed
make[2]: *** [CMakeFiles/aktualizr_static_lib.dir/src/dbusgateway/dbusgateway.cc.o] Error 1
~~~~~

Could you take a look at this one, too please?

I've now created a Dockerfile for testing against Debian Stable, so the following commands should be a good test:

    docker build -t advancedtelematic/aktualizr-stable -f Dockerfile.deb-stable  .
    docker run --rm -it advancedtelematic/aktualizr-stable src/scripts/test.sh

**Comment by cajun-rat** [comment]
Fixed in #e333d40b8887b5b88dfdf5c98ee0ec767a16dfc0 and tested in #a813df712a63821831da56ec712d62b88a27767d
